WALLET.md

```diff
@@ -117,140 +145,41 @@ bundle install
-
-2. The build process that gets executed with `yarn dev:ios` is not able to finish with `nvm` installed.
-
-3. Running the necessary scripts with an M1-native node version will not work.
-
-4. It is currently not possible to run the repository in the integrated VSCode terminal.
-
-5. It is not possible to run the repository when the containing folder is located anywhere within `Documents` or `Desktop`.
+#### Install Rosetta
 
-##### Solution
-
-1. Make sure all necessary software (VS Code, Terminal/iTerm2, XCode, Simulator) is running with Rosetta. You can verify this by typing `arch` in your terminal.
-
-```console
-$ arch
--> i386
-```
-
-2. Reinstall `brew` with x86_64 architecture:
-
-```console
-arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-```
-
-3. (optional) add your different brew installation paths as aliases to your `.zshrc`:
+If you are unable to run the app in the iOS Simulator, install Rosetta:
 
 ```bash
-alias ibrew='arch -x86_64 /usr/local/bin/brew'
-alias mbrew='arch -arm64e /opt/homebrew/bin/brew'
-```
-
-4. Install [`rbenv`](https://github.com/rbenv/rbenv) on Intel architecture:
-
-```console
-ibrew install rbenv
-```
-
-4.1. Add rbenv initialization to your shell. For instance in `.zshrc` or `.bashrc`:
-
-```bash
-eval "$(rbenv init -)"
-```
-
-5. Install ruby version 2.7.6 with `rbenv` and set it as the main version:
-
-```console
-rbenv install 2.7.6
-rbenv global 2.7.6
-```
-
-6. Install the required node version from [`.nvmrc`](/.nvmrc) with Intel architecture (If you have it installed under M1 architecture already, uninstall):
-
-```console
-(nvm uninstall 16.5.0)
-nvm install 16.5.0
-nvm use
-```
-
-Verify that node is using x64 architecture:
-
-```console
-node -e 'console.log(process.arch)'
--> x64
+/usr/sbin/softwareupdate --install-rosetta --agree-to-license
 ```
 
-7. The build script will fail if `node` + `npm` and `yarn` have been installed through `brew`. Please uninstall them through brew (the nvm installations will still be there) and install them through `nvm`:
-
-```console
-
-brew uninstall npm yarn node
-
-which node
-->  /Users/[youruser]/.nvm/versions/node/v16.15.0/bin/node
-
-nvm install-latest-npm
-which npm
--> /Users/[youruser]/.nvm/versions/node/v16.15.0/bin/npm
-
-npm install -g yarn
- which yarn
--> /Users/[youruser]/.nvm/versions/node/v16.15.0/bin/yarn
-```
-
-8. Now verify that everything is correct for the build:
-
-```console
-which ruby
--> /opt/homebrew/opt/rbenv/shims/ruby
-
-which node
--> /Users/[youruser]/.nvm/versions/node/v16.15.0/bin/node
-
-node -e 'console.log(process.arch)'
--> x64
-
-which npm
--> /Users/[youruser]/.nvm/versions/node/v16.15.0/bin/npm
-
-which yarn
--> /Users/[youruser]/.nvm/versions/node/v16.15.0/bin/yarn
-
-```
-
-Now follow [the steps for iOS installation](#install-cocopods-bundler-and-download-project-dependencies).
-
 ### Android
```

Comment:
It might be nice to have some high level Android points in the doc incase the RN site is ever down.

1. [Download and install Android Studio](https://developer.android.com/studio/index.html) and the following add-ons: Android SDK, Android SDK Platform and Android Virtual Device.
2.  Install the Android 13 (Tiramisu) SDK. It can be found can be installed through the SDK Manager in Android Studio.
3. Configure the ANDROID_HOME environment variables by adding the following lines to your `~/.zprofile` or `~/.zshrc`. You can find the actual location of the SDK in the Android Studio "Preferences" dialog, under Appearance & Behavior → System Settings → Android SDK.

```bash
export ANDROID_HOME=$HOME/Library/Android/sdk
export PATH=$PATH:$ANDROID_HOME/emulator
export PATH=$PATH:$ANDROID_HOME/platform-tools
```

Reply:
Thanks for the edit, added it! It is also more consistent with the iOS part, where we are listing the exact steps to follow.